ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -471,7 +471,6 @@ presubmits:
       ci.openshift.io/generator: cluster-init
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-release-master-build04-dry
-    optional: true
```

Comment:
Please revert this. The job has been intentionally made optional due to the cluster outage. The `build-clusters` job is also optional, you can ignore that it is failing.

Reply:
Thanks for confirming, updating this now.